src/praisonai-code/praisonai_code/cli/interactive/async_tui.py

```
@@ -1,6 +1,1866 @@
+                        # Wrapper-only optimisation; the except below falls back
+                        # to plain sync execution when it is unavailable.
+                        from praisonai_code._wrapper_bridge import import_wrapper_module
+                        run_sync_or_offload = import_wrapper_module(
+                            "praisonai._async_bridge"
+                        ).run_sync_or_offload
+                        response = run_sync_or_offload(
+                            agent.astart(prompt),
+                            thread_name="praisonai-tui-turn",
+                        )
+                        logger.debug("Used async execution (astart)")
+                    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)